share/xtp/xml/qmmm.xml

```diff
@@ -1,12 +1,28 @@
 <?xml version="1.0"?>
 <options>
   <qmmm help="Executes qmmm calculations for individual molecules and clusters" section="sec:qmmm">
-    <print_regions_pdb help="print the geometry of the regions to a pdb file" default="true" choices="bool"/>
-    <max_iterations help="max iterations for qmmm scf loop" default="100" choices="int+"/>
-    <map_file help="xml file with segment definition" default="votca_map.xml"/>
-    <job_file help="name of jobfile to which jobs are written" default="qmmm_jobs.xml"/>
-    <write_parse>
-      <states help="states to write jobs for and which to parse from jobfile" default="n e h"/>
-    </write_parse>
+    <print_regions_pdb help="print the geometry of the regions to a pdb file" default="true" choices="bool" />
+    <max_iterations help="max iterations for qmmm scf loop" default="100" choices="int+" />
+    <map_file help="xml file with segment definition" default="votca_map.xml" />
+    <job_file help="name of jobfile to which jobs are written" default="qmmm_jobs.xml" />
+    <io_jobfile help="options to write a jobfile">
+      <states help="states to write jobs for and which to parse from jobfile" default="n e h" />
+      <segments help="segments to write jobs for and which to parse from jobfile" default="all"/>
+      <use_gs_for_ex help="If true uses the ground state geometry for excited states" default="false"/>
+    </io_jobfile>
+    <regions list="" help="Definitions of all regions inside qmmmm" default="REQUIRED">
+      <qmregion default="OPTIONAL" link="region.xml" help="qmregion with dft and optionally gwbse">
```

Comment:
@JensWehner you are referring to `region.xml` here, but didn't added that file, hence the documentation build fails.

Reply:
I think I fixed this now.